app/views/helpers/returnDetailsSummary.scala.html

```diff
@@ -53,12 +53,25 @@ <h2 class="govuk-heading-m" id="lostOrDestroyed">@messages("lostOrDestroyed")</h
 <h2 class="govuk-heading-m" id="amount-to-pay-title">@AmountToPaySummary.sectionHeaderTitle(amounts.total)</h2>
 @govukSummaryList(AmountToPaySummary.amountToPaySummary(amounts))
 
+@if(userAnswers.packagingSiteList.nonEmpty || userAnswers.warehouseList.nonEmpty){
+ <h2 class="govuk-heading-m" id="registeredUkSites">@messages("registeredUkSites")</h2>
```

Comment:
Should we have 2 h2 headings in here?

Reply:
they show the same message for if either is true

Reply:
I mean could the h2 on line 57 and line 71 both show, and would that be correct?